### PR TITLE
feat(remix-dev): Add loaders config for 3d models

### PR DESCRIPTION
**Pull request by ikarus-akhil**
Adds support for importing `.glb`, `.gltf` and `.hdr` files.

Closes: #4430 

### Testing Strategy:
I was able to import `.glb` and `.hdr` files and use them in google model viewer linked using its CDN.

``` js
import model from "../../public/mod/Hololens.glb";
import env from "../../public/mod/Neutral.hdr";
const Model = () => {
  return (
    <model-viewer
      style={{ height: "100vh", width: "100vw" }}
      src={model}
      shadow-intensity="1"
      camera-controls
      touch-action="pan-y pinch-zoom"
      environment-image={env}
      camera-orbit="calc(-0.25rad + env(window-scroll-y) * 12rad) calc(90deg + env(window-scroll-y) * 45deg) calc(10m - env(window-scroll-y) * 10m)"
    />
  );
};
export default Model;
```

### CODE REVIEW

File: packages/remix-dev/modules.ts [hunk 1]
```diff
@@ -112,3 +112,15 @@ declare module "*.zip" {
   let asset: string;
   export default asset;
 }
+declare module "*.glb" {
```
Comment (machour) [inline packages/remix-dev/modules.ts:115]: Same thing here for the order please

File: packages/remix-dev/compiler/loaders.ts [hunk 2]
```diff
@@ -37,6 +37,9 @@ export const loaders: { [ext: string]: esbuild.Loader } = {
   ".woff": "file",
   ".woff2": "file",
   ".zip": "file",
+  ".glb": "file",
+  ".gltf": "file",
+  ".hdr": "file",
```
Comment (machour) [inline packages/remix-dev/compiler/loaders.ts:42]: Could you keep the list alphabetically sorted ?

**Review by machour — CHANGES_REQUESTED** [review]
Thank you for your PR 🙏
We also need you to generate the changeset, see the comment added by the bot on the PR.

**Comment by ikarus-akhil** [comment]
@machour Have made the appropriate changes.

**Comment by ikarus-akhil** [comment]
@machour Can you please review the code and approve it it is okay, as we need these loaders for our project.

**Comment by machour** [comment]
@ikarus-akhil there seem to be a conflict with the changeset file, other than that looks good to me. 👌🏼 
Someone from the Remix team will eventually attend this.

In the mean time, if you can't wait any longer, I suggest using https://www.npmjs.com/package/patch-package to apply these changes on your project.

You might also be interested in this proposal https://github.com/remix-run/remix/discussions/4549 that will turn into an RFC soon and hopefully make it to Remix.

**Comment by ikarus-akhil** [comment]
@machour  I have resolved the conflict, can you please merge this code.

**Review by machour — CHANGES_REQUESTED** [review]
Something is wrong with the changesets, it seems you edited an old one and included it again as new?

**Comment by ikarus-akhil** [comment]
@machour Sorry for the delay and all the trouble, was having problem with the changeset but now I think it should be good.

**Comment by chaance** [comment]
Looks like this was shipped in 1.11, resolved in https://github.com/remix-run/remix/pull/5030. I'm going to close this but I'll add @ikarus-akhil to our contributors list since we dropped the ball on reviewing in a timely manner. Sorry about that!

**Pull request closed without merging**